types/checkout.d.ts

```diff
@@ -1,6 +1,7 @@
 import type { PayloadWithIPOverride } from '#types/account';
 
 export type Offer = {
+  id: number | null;
```

Comment:
Why can't we just use the existing offerId property with the number as text?

Reply:
Separate from offerId, we need the actual id on InPlayer side as well, which is being sent as a param (accessFeeId) for our payment methods.
The offerId we use through the application which is combination from the actual id, but formatted as the one which is already used.